robottelo/ui/puppetclasses.py

```diff
@@ -58,3 +58,66 @@ def delete(self, name, really):
 
         self.delete_entity(name, really, locators["puppetclass.select_name"],
                            locators['puppetclass.delete'])
+
+    def import_scap_client_puppet_classes(self):
+        """Imports puppet-foreman_scap_client puppet classes."""
+        Navigator(self.browser).go_to_puppet_classes()
+        self.wait_for_ajax()
+        self.find_element(locators['puppetclass.import']).click()
+        # Checking if the scap client puppet classes are already imported
+        if self.wait_until_element(
+                locators['puppetclass.environment_default_check']):
+            self.find_element(
+                locators['puppetclass.environment_default_check']
+            ).click()
+            self.find_element(locators['puppetclass.update']).click()
+        else:
+            self.find_element(locators['puppetclass.cancel']).click()
+
+    def update_class_parameter_description(
+            self, class_name=None, parameter_name=None, description=None):
+        """Updates the description field of a given puppet class parameter."""
+        puppet_class = self.search(class_name)
+        if puppet_class is None:
+            raise UINoSuchElementError(
+                "Couldn't find the puppet class '{0}'.".format(class_name)
+            )
+        else:
```

Comment:
There's no need for the `else` clause here. If an exception is raised, the remainder of the function will fail to execute. And if someone catches the exception and handles it somehow, you probably do want to continue executing.